packages/exhibitor/buildinfo.json

```diff
@@ -3,9 +3,9 @@
   "sources": {
     "exhibitor": {
       "kind": "git",
-      "git": "https://github.com/dcos/exhibitor.git",
-      "ref": "f6654479de96e898e7ed20a5fbef6b18ac88ff24",
-      "ref_origin": "master"
+      "git": "https://github.com/rdeliallisi/exhibitor.git",
+      "ref": "df04ea8c9f9b00024dac20615c2aefd3d51aabce",
+      "ref_origin": "rdeliallisi/pattern-layout-escaped"
```

Comment:
Once this PR has two positive reviews let's merge the Exhibitor change and update this PR here to point to the master branch again.